mantle/harness/suite.go

```diff
@@ -30,6 +30,8 @@ import (
 	"sync"
 	"time"
 
+	pkgerrors "github.com/pkg/errors"
```

Comment:
Let's just drop `"errors"` instead of importing both? `pkg/errors` also has `New()`.